src/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue

```diff
@@ -114,7 +114,11 @@ export default {
             numUnclosedParens -= 1;
             // if this ")" balances out the number of "(" characters that have
             // been seen, this is the one that pairs up with the first one
-            if (openParenTokenIndex !== null && numUnclosedParens === 0) {
+            if (
+              openParenTokenIndex !== null
+              && closeParenTokenIndex == null
+              && numUnclosedParens === 0
+            ) {
               closeParenCharIndex = k;
               closeParenTokenIndex = i;
```

Comment:
Can we break this for loop early once we've found the closing parenthesis?
```suggestion
              closeParenTokenIndex = i;
              break;
```

Reply:
Yeah, I think that makes sense as an additional improvement since this inner loop doesn't need to do anything else once that closing paren has been found—good call!